container/theme.go

```diff
@@ -24,6 +24,9 @@ type ThemeOverride struct {
 // Containers will be traversed and all child widgets will reflect the theme in this container.
 // This should be used sparingly to avoid a jarring user experience.
 //
+// If the content `obj` of this theme override is a container and items are later added to the container
```

Comment:
maybe rephrase - "are later added to the container **or any sub-containers**"?

Reply:
Thanks, I pushed that in as well